script/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+if [ "$PWD" != "/var/web/mo" ]; then
+    echo Please run this from /var/web/mo.
+    exit 1
+fi
+
+if [ "$USER" != "mo" ]; then
+    echo Please run this as the mo user
+    exit 1
+fi
+
+if [ "$RAILS_ENV" != "production" ]; then
+    echo Please set RAILS_ENV to production
+    exit 1
+fi
+
+echo Going for it
+
+git pull && bundle install && rake lang:update && rake assets:precompile && rake db:migrate && uni reload
```

Comment:
I was going to add " && echo SUCCESS" to the end, but suddenly git has stopped working for me...

Reply:
Good idea.  Made the update.